src/library/basetracktablemodel.cpp

```diff
@@ -416,7 +416,9 @@ QAbstractItemDelegate* BaseTrackTableModel::delegateForColumn(
                 &BaseTrackTableModel::slotRefreshCoverRows);
         return pCoverArtDelegate;
     }
-    return nullptr;
+    // Explicitly return a TableItemDelegate, otherwise
+    // the default QStyleItemDelegate will be used.
```

Comment:
This comment does loose it's context without this PR. How about: 
```suggestion
    // Return the base class TableItemDelegate, for all other text based columns
```

Reply:
Is it possible to get did of the downcast above? 
```
auto* pTableView = qobject_cast<WTrackTableView*>(pParent);
```